src/Tribe/Repositories/Attendee/Commerce.php

```diff
@@ -14,6 +18,40 @@ class Tribe__Tickets__Repositories__Attendee__Commerce extends Tribe__Tickets__A
 	 */
 	protected $key_name = 'tribe-commerce';
 
+	/**
+	 * {@inheritdoc}
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->attendee_provider = tribe( 'tickets.commerce.paypal' );
+
+		$this->create_args['post_type'] = $this->attendee_provider->attendee_object;
+
+		// Use a regular variable so we can get constants from it in a PHP <7.0 compatible way.
+		$attendee_provider = $this->attendee_provider;
+
+		// Add object specific aliases.
+		$this->update_fields_aliases = array_merge(
+			$this->update_fields_aliases,
+			[
+				'ticket_id'       => $attendee_provider::ATTENDEE_PRODUCT_KEY,
+				'event_id'        => $attendee_provider::ATTENDEE_EVENT_KEY,
+				'post_id'         => $attendee_provider::ATTENDEE_EVENT_KEY,
+				'security_code'   => $attendee_provider->security_code,
+				'order_id'        => $attendee_provider->order_key,
+				'optout'          => $attendee_provider->attendee_optout_key,
+				'user_id'         => $attendee_provider->attendee_user_id,
+				'price_paid'      => '_paid_price',
+				'price_currency'  => '_price_currency_symbol',
```

Comment:
Why are only these two not saved as a prop?

Reply:
Updated usage